### PR TITLE
Form Completion vs Submission - Use ES instead of SQL

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke this converts this report to use ES. i didn't do too much other cleanup, but it could use some love down the line. we should definitely make it paginated... looked into doing it, but i think it's actually a decent size lift and not what i'm trying to accomplish
screen shot 2016-02-18 at 3 52 48 pm

cc: @gcapalbo

### CODE REVIEW

**Comment by benrudolph** [comment]
sadly tests did not run on that last commit...

**Comment by czue** [comment]
why aren't tests running?

File: corehq/apps/reports/standard/monitoring.py [hunk 1]
```diff
@@ -886,40 +887,43 @@ def _get_rows(self):
...
-
-            where = '(app_id, xmlns) in (%s)' % (','.join(placeholders))
-            results = FormData.objects \
-                .filter(received_on__range=(self.datespan.startdate_utc, self.datespan.enddate_utc)) \
-                .filter(user_id__in=user_map.keys()) \
-                .values('instance_id', 'user_id', 'time_end', 'received_on', 'xmlns')\
-                .extra(
-                    where=[where], params=params
-                )
-            if results.count() > 5000:
```
Comment (snopoke) [inline corehq/apps/reports/standard/monitoring.py]: Should we add a message to the UI so that the user knows that the report doesn't include all the data?

**Comment by snopoke** [comment]
:+1: will wait for build after merging #10375

File: corehq/apps/reports/standard/monitoring.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/reports/standard/monitoring.py]: it automatically does when we raise the `TooMuchDataError` which gets done here: https://github.com/dimagi/commcare-hq/pull/10368/files#diff-30c4e1ef0d3ceb995aab05246948b579R1029

**Comment by benrudolph** [comment]
:white_check_mark:

**Pull request merged**